include/pico_ws_server/web_socket_server.h

```diff
@@ -30,6 +30,11 @@ class WebSocketServer {
   // Send a BINARY message
   bool sendMessage(uint32_t conn_id, const void* payload, size_t payload_size);
 
+  // Send a TEXT message to all connections, payload must be a null-terminated string
+  bool sendMessageAll(const char* payload);
```

Comment:
nit: maybe it should be called `broadcastMessage`? This is fine if not

Reply:
I have renamed sendMessageAll to broadcastMessage now.